test/micro/org/openjdk/bench/vm/floatingpoint/DremFrem.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Oracle did not write this code so it should have your own copyright on it, unless you copied it from other OpenJDK code in which case it could have dual copyright.

Reply:
I have put there Azul now.